components/headers/ChatHeader.tsx

```diff
@@ -79,10 +97,22 @@ const ChatHeader: React.FC<ChatHeaderDataProps> = ({
 
   const closePopups = () => {
     setShowStartFlowModal(false);
+    setShowCollectionPopup(false);
     setShowChatPopup(false);
   };
 
   const popupData = getPopupData(popupTask);
+  const collectionPopupData = getCollectionPopupData(isContactType, id);
+  const optionVariables = isContactType
+    ? {
+        filter: {},
+        opts: { limit: null, offset: 0, order: 'ASC' },
+      }
+    : {
```

Comment:
```suggestion
  const optionVariables = isContactType
    ? {
        filter: {},
        opts: { limit: 50, offset: 0, order: 'ASC' },
      }
    : {
```

have a limit of 50 for now. Contacts could go up to 1 lakh. We need to make this dynamic and searchable. 
Let's put a limit for now and add the search and select functionality in a new PR

Reply:
Sure, made suggested changes.